lib/msal-browser/src/utils/BrowserUtils.ts

```diff
@@ -40,7 +40,8 @@ export class BrowserUtils {
      * Clears hash from window url.
      */
     static clearHash(): void {
-        if ("replaceState" in history) {
+        // Office.js sets history.replaceState to null
+        if (typeof history.replaceState === "function") {
```

Comment:
:+1: Maybe worth checking if there are more `in` checks in the code.

Reply:
Don't see any others.